README.md

```diff
@@ -2,31 +2,26 @@
-
-The steps for reproducing the results from the manuscript are detailed below. To retrain a new model use all steps. To analyze results from the manuscript run steps 1-3, then use steps 9-10 for analysis. Model ouptut used for the manuscript is located in `03_model/out/Manuscript_Results` 
+The steps for reproducing the results from the manuscript are detailed below. To retrain a new model use all steps. To analyze results from the manuscript run steps 1-2, then use steps 7-8 for analysis. Model ouptut used for the manuscript is located in `03_model/out/Manuscript_Results` 
 
 #### Steps for reproducibility:
 
 1) clone this repo using `git clone git@github.com:USGS-R/drb-estuary-salinity-ml.git --recurse-submodules`, the `--recurse-submodules` command initiates and updates the `river-dl` submodule housed in `03_model/src/` directory 
-2) add the file `953860.zip` into the `01_fetch/in` folder, the file can be found on S3 in drb_estuary_salinity/01_fetch/in
-3) from within the github cloned directory create the environment using `conda env create -f environment.yaml` or `conda env update --file environment.yaml –prune` if you have already created the environment and just need to update it
+2) from within the github cloned directory create the environment using `conda env create -f environment.yaml` or `conda env update --file environment.yaml –prune` if you have already created the environment and just need to update it
 
 #### To (re)train the model:  
 
-4) run `snakemake -s Snakefile_fetch_munge -j` (-j runs the job on the available number of cpus cores, use -j 2 for fewer)
-5) you might have to rerun the same command if there is an error that pops up, this is because snakemake doesn't run rules in order and some directories need to be created
-6) ensure that the `saltfront_updated.csv` is located in `03_model/in` directory
-7) now open the file `03_model/model_config.yaml` and adjust modeling parameters, and change the run_id to whatever you want to name the test run, say Test_Run
-8) run `snakemake -s Snakefile_run_ml_model run_replicates -j`, you should see the training progress in the command window and you should have model results written to `03_model/out/Test_Run/`
+3) run `snakemake -s Snakefile_fetch_munge -j` (-j runs the job on the available number of cpus cores, use -j 2 for fewer)
+4) you might have to rerun the same command if there is an error that pops up, this is because snakemake doesn't run rules in order and some directories need to be created
+5) now open the file `03_model/model_config.yaml` and adjust modeling parameters, and change the run_id to whatever you want to name the test run, for example `Test_Run`
+6) run `snakemake -s Snakefile_run_ml_model run_replicates -j`, you should see the training progress in the command window and you should have model results written to `03_model/out/Test_Run/`
```

Comment:
do you have the model output that was used in your manuscript? I think we should also add this data if it is small. This is a recommended piece of the model archives, and probably also good to include if it can't be reproduced exactly by the model.

Reply:
For the model output, is it ok to just add the model output and not the intermediate files that can be produced by the code? I'm thinking of the `inputs.zarr` and `targets.zarr` etc.

Reply:
Yes I think so. We need the inputs so it is fully reproducible. And I think we want the outputs because sometimes they can't be exactly replicated (and it might just make things easier for someone interested in the data underlying your manuscript if they don't need to rerun everything). I think everything that was just an intermediate to the end results in your paper can be left out if the code can reproduce them (at least approximately).